tests/integration/test_examples.py

```diff
@@ -4,8 +4,8 @@
 
 class TestExamples(BaseTest):
 
-    def test_files(self, app):
```

Comment:
I'm not sure why we're removing the file upload tests, I don't think they're related to the new api.